tests/core/asyncio/test_broadcast_config.py

```diff
@@ -18,7 +18,9 @@ class BroadcastEvent(BaseEvent):
     client_a.subscribe(BroadcastEvent, return_queue.put_nowait)
     client_b.subscribe(BroadcastEvent, return_queue.put_nowait)
 
-    await server.wait_until_all_remotes_subscribed_to(BroadcastEvent)
```

Comment:
E.g. this test could then stay as it was before but use

`await server.wait_until_all_remotes_subscribed_to(BroadcastEvent, exclude_self=True)`